frontend_app_html-document/src/css/index.styl

```diff
@@ -20,21 +20,25 @@
       font-size 20px
   &__contentpage, &__contentpage__edition
     width 100%
+    &__option
+      display flex
+      justify-content flex-end
     &__left
       &__wrapper
-        height 100%
+        height calc(100% - 102px)
```

Comment:
Maybe add an explanation for this `102px`: what are they corresponding to?

Reply:
fixed